lib/flight_job/models/script.rb

```diff
@@ -114,19 +108,15 @@ def exists?
     end
 
     def notes
-      @notes ||= if File.exist? notes_path
-                   File.read notes_path
-                 else
-                   ''
-                 end
+      @notes ||= Notes.new(id).tap { |n| n.read }
```

Comment:
I'm not keen on automatically reading the notes here.  Firstly `Notes.new(id)` just looks much nicer than `Notes.new(id).tap { |n| n.read }`; secondly reading the notes from disk into memory is potentially a memory hungry and time consuming process.  Especially when dealing with multiple scripts.

I'm not sure that we needlessly call the `notes` method at the moment, but given reading the notes requires `notes.read` having them automatically read seems awkward.

IIUC, they are read here to ensure that they are set to a string.  If they aren't saving the notes to disk fails.  I think that this can be better handled by doing the following in `Notes#save`:

```ruby
File.write path, @notes || ''
```